### ISSUE TITLE
in vaxjo we have only 5 schedule per day is it ok to book

### ISSUE THREAD

**Issue by ecreation16** [lopotras/WoltPartner-Schedule#7]

**Comment by ecreation16** [comment]
is this programme running or not,  

https://user-images.githubusercontent.com/80490925/110871321-7869b700-82ce-11eb-8937-7a7a7df4ba71.mp4


after running the code window is coming and stop, I think what is the issue. for our schedule only available 5 is it a problem

**Comment by tdek** [comment]
you have to activate phyton first. to do this go on https://conda.io/activation and then follow the instructions :) 

I had to do that too!

**Comment by lopotras** [comment]
@ecreation16 let me know if that worked.

Also can you tell me on what days and times do your shifts show up? And for each time what days do the shifts show up for?

**Comment by ecreation16** [comment]
@tdek 
may I follow all step in that web page instruction

**Comment by ecreation16** [comment]
@lopotras 

Yes Thursday 1600 Friday 09:00 we have only booking list 5 items as in my video, still same issue i want to know can i run this code in anaconda navigation - Jupiter notebook, i try it but it not work...can i run in that soft ware.

**Comment by tdek** [comment]
@ecreation16 Yes you have to follow the instructions on that website. I did that too and fixed the issue. 

You can’t run the command if phyton is not activated.

**Comment by ecreation16** [comment]
@lopotras
Hi thank you very much i fix my running error but my booking time are thrursday 1600 pm and Friday 0900 am can you change the auto run code time and send me the script file, because when I run it at fix time 1600 pm it have some difference between wolt partner app and my computer time.

**Comment by ecreation16** [comment]
@lopotras Or can you make the code run 2 min with refreshing thank you very much....

**Comment by tdek** [comment]
@ecreation16 
The way to address the shifts appearing at 16.00 and 09.00 is provided in Advanced Tips in the Readme file.

You can change that and then the Programme will start booking shifts for you at that time. 

You have to open up the Wolt courier app in bluestacks and go to the free scheduled hours in your city. 
Don’t move your mouse. 

The Programme will constantly refresh the page and try to book the shifts that you selected. 

At the same time you can try to book the shifts on your phone 

Hope I could help you. 

If there’s a time difference between your phone and your pc, you have to change the time on your pc, so that it starts booking when it should. It depends on that time.

**Comment by lopotras** [comment]
@ecreation16 did that solve the issue?

**Comment by ecreation16** [comment]
@lopotras  no software is working now but it not book the time. Software is running but it not book the time

**Comment by ecreation16** [comment]
the software stopped at refresh step nothing booking function is run...see the video...

**Comment by ecreation16** [comment]
https://user-images.githubusercontent.com/80490925/112244263-4f8edd80-8c4f-11eb-9d1c-cd99c29a6aa6.mp4

booking action run after one or 2 minute

**Comment by lopotras** [comment]
Can you tell which days' shifts show up for which days? 
(e.x. in Aarhus the shifts show up on Thursday at 15:00 for next Friday-Sunday and on Friday at 08:00 for next Monday-Thursday)

This way I will be able to update the code generator to fit your times.

**Comment by ecreation16** [comment]
Thursday 16.00 ( Friday, Saturday, sunday )  
Friday 09.00 ( Monaday,Tuesday,Wednesday,Thursday )
yes may be that is the issue Please update it and give me the new one...it is really appreciable

**Comment by lopotras** [comment]
Is the script running correctly on Thursdays and Fridays?

In the video you sent the program is run on Wednesday though. It does not book any shifts, because no new days actually show up (according to the Thursday/Friday model of new shifts coming).

Do you have also shifts coming in on Wednesday that you want to be able to book? Is it just that one shift for the upcoming Sunday?

**Comment by ecreation16** [comment]
no i did that video at the night i cancel my booking to check that the programmed is running at the time Thursday and friday but after refreshing it not book the times times available but it not book the times...
